modules/EmbedPlayer/resources/mw.EmbedTypes.js

```diff
@@ -13,7 +13,15 @@
  * loaded post player detection
  */
 //Native Mobile player
-var nativeComponentPlayerVideo = new mw.MediaPlayer( 'nativeComponentPlayer', ['video/h264', 'video/mp4', 'application/vnd.apple.mpegurl', 'video/wvm'], 'NativeComponent' );
+var nativeFormats = window.kNativeSDK ? window.kNativeSDK.supportedFormats : null;
+if (nativeFormats) {
+    // Extract mimetypes from keys. 
+    nativeFormats = Object.keys($.extend({}, nativeFormats.drm, nativeFormats.clear));
```

Comment:
@noamtamim  -  you are doing the same object/keys/merge 3 times - maybe save it to the kNativeSDK object? and re-use it?